terraform/modules/gce/compute/nodejs/nodejs.tf

```diff
@@ -0,0 +1,227 @@
+#--------------------------------------------------------------
+# This module creates all resources necessary for the
+# Node.js application
+#--------------------------------------------------------------
+
+variable "name"                { default = "nodejs" }
+variable "region"              { }
+variable "vpc_id"              { }
+variable "vpc_cidr"            { }
+variable "key_name"            { }
+variable "azs"                 { }
+variable "private_subnet_ids"  { }
+variable "public_subnet_ids"   { }
+variable "site_ssl_cert"       { }
+variable "site_ssl_key"        { }
+variable "vault_ssl_cert"      { }
+variable "atlas_username"      { }
+variable "atlas_environment"   { }
+variable "atlas_aws_global"    { }
+variable "atlas_token"         { }
+variable "blue_ami"            { }
+variable "blue_nodes"          { }
+variable "blue_instance_type"  { }
+variable "blue_weight"         { }
+variable "green_ami"           { }
+variable "green_nodes"         { }
+variable "green_instance_type" { }
+variable "green_weight"        { }
+variable "sub_domain"          { }
+variable "route_zone_id"       { }
+variable "vault_token"         { default = "" }
+variable "vault_policy"        { default = "nodejs" }
+
+resource "google_compute_firewall" "nodejs" {
+  name        = "${var.name}"
+  network     = "${var.network}"
+  description = "Firewall rule for Nodejs"
+
+  tags      { Name = "${var.name}-elb" }
+  lifecycle { create_before_destroy = true }
+
+  allow {
+    protocol = "tcp"
+    ports    = ["80", "443"]
+  }
+
+  allow {
+    protocol    = "icmp"
+    //TODO add tags to subnets
+  }
+}
+
+resource "aws_elb" "blue" {
+  name                        = "${var.name}-blue"
+  connection_draining         = true
+  connection_draining_timeout = 400
+
+  subnets         = ["${split(",", var.public_subnet_ids)}"]
+  security_groups = ["${aws_security_group.elb.id}"]
```

Comment:
Don't think this resource exists in this module, so maybe just remove all ELB's.